src/forms/Textarea.jsx

```diff
@@ -139,9 +120,6 @@ export class Textarea extends React.PureComponent<Props, State> {
 					required={required}
 					onChange={this.onChange}
 					rows={rows}
-					ref={textarea => {
```

Comment:
Hmmm, does removing the ref break expected behavior in mup-web? Not quite sure how we would know this, but I remember in some components ref's were manually talked in like this intentionally.

Reply:
This ref is only used to as a way to pass a dom node to autosize, which calculates size of the texarea. It'll break the functionality but this should be added to the swarm-components version. Right now the refs wouldn't be forwarded since this is a swarm component and not the `textarea` node.